tests/unit/lms/resources/_js_config_test.py

```diff
@@ -47,16 +47,30 @@ def test_it(
         )
         assert authToken == bearer_token_schema.authorization_param.return_value
 
-    def test_it_is_None_for_non_lti_users(self, context, pyramid_request):
-        pyramid_request.lti_user = None
-
-        assert JSConfig(context, pyramid_request).config["authToken"] is None
+    @pytest.mark.usefixtures("no_lti_user")
+    def test_it_is_None_for_non_lti_users(self, authToken):
+        assert authToken is None
 
     @pytest.fixture
     def authToken(self, config):
         return config["authToken"]
 
 
+class TestJSConfigDebug:
+    """Unit tests for the "debug" sub-dict of JSConfig."""
+
+    def test_it_contains_debugging_info_about_the_users_role(self, config):
+        assert "role:learner" in config["tags"]
+
+    @pytest.mark.usefixtures("no_lti_user")
```

Comment:
:+1: Having just read about what this is, I think there are a ton of places where we could benefit from using `usefixtures` more. We quite often include fixtures in the args just to make something happen which can lead to very long argument lists.

Reply:
It _should_ theoretically trigger a pylint "unused argument" warning when we do that unless we're not using pylint (as is the case in h, for example) or we've disabled that warning for the tests